src/components/AdminSettings.vue

```diff
@@ -10,6 +10,12 @@
 		</h2>
 		<div id="openai-content">
 			<div>
+				<NcNoteCard v-if="!state.assistant_enabled" type="warning">
+					{{ t('integration_openai', 'The Assistant app is not enabled. You need it to use the features provided by the OpenAI/LocalAI integration app.') }}
+					<a class="external" href="https://apps.nextcloud.com/apps/assistant" target="_blank">
```

Comment:
It might be nicer to link to the assistant app in the apps page in nextcloud instead of apps.nc.com

Reply:
Done